README.md

```diff
@@ -46,17 +46,12 @@ For bootstrapping additionally:
 - install(1) - GNU coreutils
 - objcopy(1), objdump(1), strip(1): binutils
 
-`xbps-src` requires a utility to chroot and bind mount existing directories
+`xbps-src` requires [a utility to chroot](#chroot-methods) and bind mount existing directories
```

Comment:
"a chroot-like utility" maybe? or `chroot(1)`

Reply:
sentence reads "utility to chroot and bind mount existing directories", hard to rewrite